compiler/src/main/scala/org/scalajs/core/compiler/GenJSCode.scala

```diff
@@ -5573,4 +5576,6 @@ abstract class GenJSCode extends plugins.PluginComponent
 
     case class GlobalScope(pos: Position) extends MaybeGlobalScope
   }
+
+  object UndefinedParam
```

Comment:
I think this deserves a comment :) Something like: "Marker object for undefined parameters in JavaScript semantic calls. To be used inside a js.Transient node".

Also make this a case object?